images/nginx/run.sh

```diff
@@ -37,6 +37,10 @@ if [ "$FRAME_PROTECTION_ENABLED" = "false" ]; then
    grep -v "Content-Security-Policy" /rw.mount/nginx/default.conf.tmp > /rw.mount/nginx/defaultWithoutProtection.conf.tmp
    mv /rw.mount/nginx/defaultWithoutProtection.conf.tmp /rw.mount/nginx/default.conf.tmp
 fi
+if [ "$IPV4_ONLY" = "true" ]; then
+   grep -v "# feature-ipv6" /rw.mount/nginx/default.conf.tmp > /rw.mount/nginx/defaultWithoutIPv6.conf.tmp
```

Comment:
Where this comment `# feature-ipv6` could come from as it is not in default nginx configuration, neither in portal or console images ?

```bash
docker run --rm -ti --entrypoint /bin/sh graviteeio/nginx:1.27.4 -c 'grep ipv6 /etc/nginx/conf.d/default.conf'        

docker run --rm -ti --entrypoint /bin/sh graviteeio/apim-portal-ui:4.6.4 -c 'grep ipv6 /etc/nginx/conf.d/default.conf'   

docker run --rm -ti --entrypoint /bin/sh graviteeio/apim-management-ui:4.6.4 -c 'grep ipv6 /etc/nginx/conf.d/default.conf'
```

Not found anywhere 🤔

Reply:
Ok, I don't see the same thing in [the commit](https://github.com/gravitee-io/gravitee-api-management/pull/10178/commits/18dae5336dd97a16bc2957565c06204f2d0b27ec) but I got the point.

Could we avoid a match on a comment and remove the match on the real value `listen [::]:` :pray:  ?